config/manager/manager.yaml

```diff
@@ -31,7 +31,7 @@ spec:
         - /manager
         args:
         - --leader-elect
-        imagePullPolicy: Always
+        imagePullPolicy: IfNotPresent
```

Comment:
Are there concerns with this change?

Without it the step that loads the local image into the kind cluster is redundant because it just attempts to pull regardless. If this isn't an acceptable change another possibility could be to push the built image to a registry (as discussed and intentionally avoided previously).

@toddbaert @beeme1mr @AlexsJones @james-milligan

Reply:
I think in any non-dev scenario, the images would have a real version appended, so I think this is fine. If anything it could impact local dev a bit, but with the safeguards this PR adds I think that's worth it.